src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -0,0 +1,280 @@
+<template>
+  <div>
+    <p>
+      Add the people and corporations/firms who will have a role in your company. People can have multiple roles;
+      Corporation/firms can only be Incorporators.
+    </p>
+    Your application must include the following:
+    <ul>
+      <li>
+        <v-icon color='blue' v-if="hasRole(Roles.COMPLETING_PARTY, 1, 'EXACT')" class="cp-valid">mdi-check</v-icon>
+        <span v-else><v-icon color='red' v-if="showErrors" class="cp-invalid">mdi-close</v-icon></span>
+        <span class='chk-list-item-txt'>The Completing Party</span>
+      </li>
+      <li>
+        <v-icon color='blue' v-if="hasRole(Roles.INCORPORATOR, 1, 'ATLEAST')" class="incorp-valid">mdi-check</v-icon>
+        <span v-else><v-icon color='red' v-if="showErrors" class="incorp-invalid">mdi-close</v-icon></span>
+        <span class='chk-list-item-txt'>At least one Incorporator</span>
+      </li>
+      <li>
+        <v-icon color='blue' v-if="hasRole(Roles.DIRECTOR, 1, 'ATLEAST')" class="dir-valid">mdi-check</v-icon>
+        <span v-else><v-icon color='red' v-if="showErrors" class="dir-invalid">mdi-close</v-icon></span>
+        <span class='chk-list-item-txt'>At least one Director</span>
+      </li>
+    </ul>
+    <div class="btn-panel" v-if="orgPersonList.length === 0">
+      <v-btn outlined color="primary" @click="addOrgPerson([ Roles.COMPLETING_PARTY ], IncorporatorTypes.PERSON)"
+      :disabled="showOrgPersonForm" id="btn-start-add-cp">
+        <v-icon>mdi-account-plus-outline</v-icon>
+        <span>Start by Adding the Completing Party</span>
+      </v-btn>
+    </div>
+    <div class="btn-panel"  v-if="orgPersonList.length > 0">
+      <v-btn outlined color="primary" @click="addOrgPerson([], IncorporatorTypes.PERSON)"
+      :disabled="showOrgPersonForm" id="btn-add-person">
+        <v-icon>mdi-account-plus</v-icon>
+        <span>Add a Person</span>
+      </v-btn>
+      <v-btn outlined color="primary" :disabled="showOrgPersonForm" class="spacedButton"
+      @click="addOrgPerson([Roles.INCORPORATOR], IncorporatorTypes.CORPORATION)" id="btn-add-corp">
+        <v-icon>mdi-domain-plus</v-icon>
+        <span>Add a Corporation or Firm</span>
```

Comment:
Per mockup, please add handling for BCOMP vs COOP (ie, different label).

Reply:
Found this, updated!